### PR TITLE
[16.0][MIG] website_sale_product_attribute_filter_collapse: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT44382

@chienandalu @CarlosRoca13 please review :)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
/ocabot migration website_sale_product_attribute_filter_collapse

File: website_sale_product_attribute_filter_collapse/views/templates.xml [hunk 1]
```diff
@@ -5,112 +5,60 @@
     <template
         id="products_attributes_collapsible"
         inherit_id="website_sale.products_attributes"
-        name="Collapsible attribute filters"
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_collapse/views/templates.xml]: Why removing this?

File: website_sale_product_attribute_filter_collapse/views/templates.xml [hunk 2]
```diff
@@ -5,112 +5,60 @@
...
-        active="False"
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_collapse/views/templates.xml]: Well, this changes the default behavior of the module. You should keep it IMO

**Review by chienandalu — CHANGES_REQUESTED** [review]

File: website_sale_product_attribute_filter_collapse/views/templates.xml (hunk 2 above)
Comment (pilarvargas-tecnativa) [reply website_sale_product_attribute_filter_collapse/views/templates.xml]: I was adapting the configuration to the web editor but I deleted more than I should have. It is now restored as it was and the behaviour has been corrected to be the same. thanks

**Review by chienandalu — APPROVED** [review]
Now is great :slightly_smiling_face: :+1:

File: website_sale_product_attribute_filter_collapse/__manifest__.py [hunk 3]
```diff
@@ -3,14 +3,15 @@
 {
     "name": "Website Sale Attribute Filter Collapse",
     "summary": "Allows the attributes of the categories to be folded",
-    "version": "15.0.1.1.1",
+    "version": "16.0.1.1.1",
```
Comment (CarlosRoca13) [inline website_sale_product_attribute_filter_collapse/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by CarlosRoca13 — COMMENTED** [review]
Please set correct version

File: website_sale_product_attribute_filter_collapse/__manifest__.py (hunk 3 above)
Comment (pilarvargas-tecnativa) [reply website_sale_product_attribute_filter_collapse/__manifest__.py]: thanks :)

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**